vendor/gopkg.in/yaml.v2/decode.go

```diff
@@ -315,6 +319,13 @@ func (d *decoder) prepare(n *node, out reflect.Value) (newout reflect.Value, unm
 }
 
 func (d *decoder) unmarshal(n *node, out reflect.Value) (good bool) {
+	d.decodeCount++
```

Comment:
@kolyshkin not critical, but it probably won't hurt to have; the beef of the change is in this file (although their fix is a bit weird)